src/http-client.js

```diff
@@ -271,10 +283,11 @@ export default opts => {
     publicRequest: (method, url, payload) => pubCall(url, payload, method),
 
     order: payload => order(privCall, payload, '/api/v3/order'),
+    orderOco: payload => orderOco(privCall, payload, '/api/v3/order/oco'),
     orderTest: payload => order(privCall, payload, '/api/v3/order/test'),
     getOrder: payload => privCall('/api/v3/order', payload),
     cancelOrder: payload => privCall('/api/v3/order', payload, 'DELETE'),
-
+  
```

Comment:
Could you remove this?

Reply:
Oops, my diff tool must have been ignoring whitespace. 🤭Removed.